packages/integration-tests/test/circuit-relay.node.ts

```diff
@@ -352,45 +326,32 @@ describe('circuit-relay', () => {
     beforeEach(async () => {
       [local, remote, relay1, relay2, relay3] = await Promise.all([
         createClient({
-          transports: [
-            tcp(),
-            circuitRelayTransport({
-              discoverRelays: 3
-            })
-          ]
-        }),
-        createClient({
-          transports: [
-            tcp(),
-            circuitRelayTransport({
-              discoverRelays: 1
-            })
-          ]
+          addresses: {
+            listen: [
+              '/p2p-circuit',
+              '/p2p-circuit',
+              '/p2p-circuit'
```

Comment:
This feels more odd than `discoverRelays: 3`

Reply:
Tbh there’s not a great reason to listen on more than one relay and a number of downsides to it - bigger identify messages, bigger provider records/peer-info objects.

Also given that the number of browser nodes can vastly outnumber publicly dialable server nodes it’s probably not something we should encourage anyway as a small number of relay clients could end up consuming all available relay slots on the network.